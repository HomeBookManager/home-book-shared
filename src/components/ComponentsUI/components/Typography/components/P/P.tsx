import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TypographyFontType } from '../../enums';
import { TOmittedTypographyProps } from '../../types';

const P: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.p} {...restProps}>
    {children}
  </Typography>
);

export default P;
