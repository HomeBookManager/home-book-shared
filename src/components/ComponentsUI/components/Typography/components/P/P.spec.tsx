import { render } from '@testing-library/react';

// components
import P from './P';

describe('P snapshots', () => {
  it('should render P', () => {
    //before
    const { asFragment } = render(<P>P</P>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
