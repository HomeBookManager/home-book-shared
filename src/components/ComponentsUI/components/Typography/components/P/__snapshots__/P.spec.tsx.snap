// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`P snapshots should render P 1`] = `
<DocumentFragment>
  <p
    class="Typography Typography--regular Typography--normal Typography--p"
    data-test-text=""
  >
    P
  </p>
</DocumentFragment>
`;
